packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -186,11 +198,12 @@ class Editor extends React.Component {
 
   getLinkedStyles = () => {
     const theme = this.context;
+    const { controlPosition } = this.props;
 
     return css.resolve`
       .wrapper {
         background-color: ${theme.palette.common.white};
-        border: 1px solid ${theme.palette.grey[400]};
+        border: 1px solid #979797;
```

Comment:
doesnt exist anywhere in the UI Kit 🙈

Reply:
Ya this is slightly annoying. Is there any color that even comes close? I would rather use a "close" color that is in the kit than a random hard coded color.